Formula/n/newnode-helper.rb

```diff
@@ -0,0 +1,55 @@
+class NewnodeHelper < Formula
+  desc "Web proxy that uses a distributed p2p network to circumvent censorship"
+  homepage "https://www.newnode.com/newnode-vpn"
+  url "https://github.com/clostra/newnode.git",
+    tag: "2.1.5", revision: "18a3f713267b2a08e34cbe04253df891889997a2"
+  license "GPL-2.0-only"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "coreutils" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on xcode: ["9.3", :build]
+  depends_on :macos
+
+  def install
+    system "./build.sh"
+    bin.install "client" => "newnode-helper"
```

Comment:
Why not keep upstream naming?

Reply:
It might clash with another project, a messenger based on this vpn. Would it be okay to leave it like this?

Reply:
Is that in Homebrew/core?

Reply:
Casks are a very different type. I don't think there is any reason to rename this at present

Reply:
I've changed the name to match the upstream